e2e_tests/test_suites/patch/test_setup.go

```diff
@@ -189,6 +205,15 @@ func aptHeadImageTestSetup() []*patchTestSetup {
 	return imageTestSetup(mapping)
 }
 
+func aptDownradeImageTestSetup() []*patchTestSetup {
+	// This maps a specific patchTestSetup to test setup names and associated images.
+	mapping := map[*patchTestSetup]map[string]string{
+		aptDowngradeSetup: utils.DowngradeAptImages,
```

Comment:
just use HeadAptImages here?

Reply:
I haven't tested how adding other debians than 10 works for this test I am pinning a package from a certain date that was meant for debian 10

Also HeadAptImages have ubuntus

Reply:
But this is an apt change test, we should test all apt images